tests/ci_visibility/api/test_internal_test_visibility_api.py

```diff
@@ -44,76 +44,81 @@ def test_api_is_item_itr_skippable_false_when_skipping_disabled_test_level(self)
             itr_enabled=True,
             skipping_enabled=False,
             suite_skipping_mode=False,
-            skippable_items={
-                "skippable_module/suite.py": ["skippable_test"],
-            },
+            skippable_items=_make_fqdn_test_ids([("skippable_module", "suite.py", "skippable_test")]),
         ):
-            CIVisibility.enable()
+            try:
```

Comment:
nit: Should we use a try/finally in a test? I guess there's no equivalent of `tearDown()` in pytest, so this is probably the cleanest way 🤔

Reply:
We could use a a fixture but I've got vague plans to rewrite the entire test setup/teardown fixture to be more pleasant than this so I'm going to leave it as is for now.